### DISCUSSION TITLE
Moving plots to another machine (different operating system)

### DISCUSSION THREAD

**Discussion by leoliyuanduo** [Chia-Network/chia-blockchain#1975] [Q&A]
Can I move plots to another machine using different operating system? like using mac os potting to hhd hard drive and start using a windows system computer to farming?
Thanks.

**Comment by hoffmang9** [comment]
https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#can-i-make-plots-on-one-machine-and-move-it-to-another-machine
